report_pre_thesis/src/sections/methodology.tex

```diff
@@ -1,8 +1,246 @@
+This results in two separate datasets, each normalized in a distinct manner but derived from the same original dataset.
+ICA and PLS-SM are then performed on each of these datasets separately.
+The reason for this is that, as \citet{cleggRecalibrationMarsScience2017} found, some of the oxides are better modeled with data normalized using Norm1, while others are better modeled with data normalized using Norm3.
+
+The pipeline for the shared preprocessing steps can be seen in figure \ref{fig:shared_preprocessing}.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.2675\textwidth]{images/shared_preprocessing.png}
+	\caption{The preprocessing steps shared by both the PLS1-SM and ICA phases of the pipeline.}
+	\label{fig:shared_preprocessing}
+\end{figure}
+
+\subsection{PLS1-SM}\label{sec:methodology_pls1}
+This section describes the PLS1-SM aspect of the pipeline.
+
+\subsubsection{Data Preprocessing}\label{sec:pls1_data_preprocessing}
+After the shared preprocessing steps described in section \ref{sec:shared_preprocessing}, the data undergoes a further step of averaging.
+Recall that each target is shot at five different locations, and each location is shot at 50 times.
+As mentioned in section \ref{sec:shared_preprocessing}, the first five shots are removed, leaving 45 shots per location.
+For PLS1-SM, the 45 shots from each location are averaged, resulting in a single spectrum per location with a total of five spectra per target.
+
+\subsubsection{Outlier Removal with Mahalanobis Distance and Chi-Squared Test}
+% General programming language differences and implementation details
+% Differences in outlier detection (automatic/quantitative vs manual/qualitative)
+% Missing information about how three of the oxides are weighted in MOC result
+% No manual cross validation in PLS
+% Holdout split 80/20
+% No inverse IRF weighing weighting in ICA
+% Missing information about outlier removal in ICA - therefore we decided to not do it
+% Missing information about how ICA uses each of the five datasets for each target
+% How we do regression in ICA (since there are not enough details about it in the papers)
+
+As mentioned in section \ref{sec:outlier_removal}, if the Mahalanobis distances can be shown to follow a chi-squared distribution, then the chi-squared test can be used to determine whether a sample is an outlier.
+Since we do not have the expertise to make a qualitative assessment of the outliers, we have instead decided to use this property of the Mahalanobis distances to automatically detect outliers.
+This works by computing the Mahalanobis distance for each data point in the training set, followed by a comparison of these distances against a chi-squared distribution.
+A data point is classified as an outlier if its Mahalanobis distance corresponds to a chi-squared statistic that exceeds the critical value of the chi-squared distribution, which is determined by the specified degrees of freedom $df$.
```

Comment:
Changed it to Nu based on the following:

> In equations, the typical symbol for degrees of freedom is ν (lowercase [Greek letter nu](https://en.wikipedia.org/wiki/Nu_(letter))). In text and tables, the abbreviation "d.f." is commonly used. [R. A. Fisher](https://en.wikipedia.org/wiki/Ronald_A._Fisher) used n to symbolize degrees of freedom but modern usage typically reserves n for sample size.

I don't think we should remove it since we do use it. And we might end up mentioning it later too (for example in experiments). Also I think this line would end up phrased weirdly if we removed it:

> Since we have leverage and spectral residuals as our two dimensions, we have $\nu = 2$, because the number of dimensions is equal to the number of degrees of freedom\cite{aggarwal_outlier_2017}.